.github/workflows/trivy-docker.yaml

```
@@ -25,15 +25,15 @@ on:
   workflow_dispatch:
 
 permissions:
-  actions: none
+  actions: write
```

Thanks to @jawnsy, we don't need this. 

```suggestion
  actions: none
```